### ISSUE TITLE
Event display and histogramming in search - need a category for them?

### ISSUE THREAD

**Issue by katilp** [cernopendata/opendata.cern.ch#1781]
Can CMS event display and online histogramming be found in search?
They should. Do they belong to any category, if not they could be "Online tools"
@sefeg @daslerr @tiborsimko What do you think?

**Comment by sefeg** [comment]
I guess that there would have to be a record describing the tool and linking to it?

Like _[Explore CMS open data and visualise detector events](http://opendata-dev.web.cern.ch/record/58)_ (that I found searching for CMS event display). There is also [iSpy-WebGl](http://opendata-dev.web.cern.ch/record/555). The second is assigned to the collection "CMS-Tools"

**Comment by katilp** [comment]
@sefeg What users would expect, is a direct link to these tools. These two records are not where 99.99999% users want to land. If we can avoid intermediate clicking for these online resources, it would be much better. Would it possible to add a category of their own for them?

**Comment by sefeg** [comment]
I agree! I am just not sure how we can show a direct "preview" of the Event display page in the search results. But for that, @tiborsimko is much better suited to comment.
